api/proto/job/job.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+option go_package = "teleport-job-worker/api/proto/job";
+package job;
+
+message StartRequest {
+  job.Command command = 1;
+  job.Resources resource_limits = 2;
+}
+
+message StopRequest {
+  string id = 1;
+}
+
+message QueryRequest {
+  string id = 1;
+}
+
+message OutputRequest {
+  string id = 1;
+}
+
+message Resources {
+  uint64 memory_bytes = 1; // Amount of memory in bytes that a job can use
+  int32 cpu_percentage = 2; // Percentage of CPU that a job can use
+  int32 disk_io_bps = 3; // Bytes per second a job can use on a disk; this value is applied separately to writes and reads
+}
+
+message Response {
+  job.Info info = 1;
+  job.Resources resource_limits = 2;
+}
+
+message OutputResponse {
+  bytes stdout = 1;
+  bytes stderr = 2;
+}
+
+message Info {
+  string id = 1;
+  job.Status status = 2;
+  uint64 start_time = 3; // Time the job started in number of milliseconds since the epoch
+  uint64 finish_time = 4; // Time the job stopped in number of milliseconds since the epoch
+  job.Command command = 5;
+  string status_info = 6; // Information about the current job status; may be empty
```

Comment:
It would provide more information about a job's status as needed. For example, if a job's state is `failed` the `status_info` field might have some human-readable string like `exceeded memory limit`.

Reply:
Gotcha. The name is okay. It could be helpful to add this context to the comments.